packages/components/src/product-icon/config.js

```diff
@@ -18,14 +24,20 @@ import wpcomPremium from './images/wpcom-premium.svg';
 
 export const paths = {
 	'jetpack-scan': jetpackScan,
+	'jetpack-scan-v2': jetpackScanV2,
 	'jetpack-backup-daily': jetpackBackup,
 	'jetpack-backup-realtime': jetpackBackup,
+	'jetpack-backup-v2': jetpackBackupV2,
 	'jetpack-free': jetpackFree,
 	'jetpack-personal': jetpackPersonal,
 	'jetpack-premium': jetpackPremium,
 	'jetpack-professional': jetpackProfessional,
+	'jetpack-all-in-v2': jetpackAllInV2,
```

Comment:
We might have to come back to this one later and change it to Complete if we stick with the latter as the final name of the bundle.

Reply:
Absolutely. I believe the deadline to settle on a name is tomorrow. I'll keep an eye on it.